dashboard/test/lib/pd/payment/payment_calculator_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+module Pd::Payment
+  class PaymentCalculatorTest < ActiveSupport::TestCase
+    self.use_transactional_test_case = true
+
+    setup_all do
+      @regional_partner = create :regional_partner
+
+      @csf_workshop = create(:pd_ended_workshop, course: Pd::Workshop::COURSE_CSF, enrolled_and_attending_users: 20)
```

Comment:
This one is only used in a single test case. setup_all is being underutilized here. If this is not shared in at least 2 test cases, what's the gain of constructing it in setup?